src/permission/fs_permission.cc

```diff
@@ -116,9 +117,15 @@ namespace permission {
 
 // allow = '*'
 // allow = '/tmp/,/home/example.js'
-void FSPermission::Apply(const std::string& allow, PermissionScope scope) {
+void FSPermission::Apply(
+    const std::string& allow,
+    PermissionScope scope,
+    const std::unordered_map<std::string, std::string>& options) {
   using std::string_view_literals::operator""sv;
-  for (const std::string_view res : SplitString(allow, ","sv)) {
+  std::string delimiter = options.find("delimiter") != options.end()
+                              ? options.at("delimiter")
+                              : ",";
```

Comment:
technically, delimiter will always have a value. I think you can just `options.at("delimiter")`